CONTRIBUTING.md

```diff
@@ -64,5 +76,5 @@ Then you just add a line to every git commit message:
 
 Use your real name (sorry, no pseudonyms or anonymous contributions.)
 
-If you set your `user.name` and `user.email` git configs, you can sign your
-commit automatically with `git commit -s`.
+If you set your `user.name` and `user.email` in your git config, you can
+sign your commit automatically with `git commit -s`.
```

Comment:
Nit: I think it's `sign-off`. Sign implies the use of PGP keys for verified commits.